src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/SDSEarlyReleaseDefaultingRulesService.kt

```diff
@@ -20,12 +20,7 @@ class SDSEarlyReleaseDefaultingRulesService(
   val extractionService: SentencesExtractionService,
   val trancheConfiguration: SDS40TrancheConfiguration,
 ) {
-
```

Comment:
requiresRecalculation seemes a bit abstract so just used and it was just passing through to another method hasAnySDSEarlyRelease